slick.grid.js

```diff
@@ -534,14 +543,19 @@ if (typeof Slick === "undefined") {
         // be enabled there so that editors work as expected); note that
         // selection in grid cells (grid body) is already unavailable in
         // all browsers except IE
-        disableSelection($headers); // disable all text selection in header (including input and textarea)
+        disableSelection(_headers); // disable all text selection in header (including input and textarea)
 
         if (!options.enableTextSelectionOnCells) {
           // disable text selection in grid cells except in input and textarea elements
           // (this is IE-specific, because selectstart event will only fire in IE)
-          $viewport.on("selectstart.ui", function (event) {
-            return $(event.target).is("input,textarea");
-          });
+          _viewport.forEach((view) => {
```

Comment:
same as earlier comment, can we keep an ES5 syntax `function () {}` since we aren't transpiling our code, there a few of these in this file, they could all be changed at same time

Reply:
Done